app/models/miq_queue_worker_base/runner.rb

```diff
@@ -101,8 +119,9 @@ def deliver_queue_message(msg, &block)
   end
 
   def deliver_message(msg)
-    return deliver_queue_message(msg) if msg.kind_of?(MiqQueue)
-    return process_message(msg)       if msg.kind_of?(String)
+    return deliver_queue_message(msg)         if msg.kind_of?(MiqQueue)
```

Comment:
does this want to be a `case` statement on `msg` / kind?
the error case being an `else`

Reply:
:+1: love it